impeller/entity/entity_pass.cc

```diff
@@ -617,6 +617,9 @@ EntityPass::EntityResult EntityPass::GetEntityForElement(
       return EntityPass::EntityResult::Skip();
     }
 
+    // Preserve the subpixel offset in the origin to position
+    // the local and global coordinates below.
+    auto origin = subpass_coverage->GetOrigin();
```

Comment:
What about capturing the roundout value into a new Rect rather than overwriting the original subpass_coverage and then each of the following uses of the coverage can decide whether they want to be based on the logical coverage or the pixel coverage.

Reply:
That seems like a good idea